### ISSUE TITLE
Unable to display raw data for tables with lots of columns

### ISSUE THREAD

**Issue by danje** [metabase/metabase#3205]
-  I am using the **Chrome** browser.
-  My computer's OS is **Windows 7 Professional SP1**.
-  I'm running Metabase **locally from the JAR** and it is using the **H2 (default)** database.
-  My data is in a **SQL Server** database.
-  My Metabase version is **0.19.3**.

We have a data warehouse with a few tables that have a huge number of columns - one of them has 657! If I try to view the raw data (even if I set a filter so that only 1 row is returned), Metabase never returns a result - it just sits there "doing science" ;-)

![property2](https://cloud.githubusercontent.com/assets/6749519/17694545/437d07e6-639c-11e6-8a80-883e02fb0912.png)

If I have a look at the JavaScript console I can see 2 error messages:

```
Uncaught (in promise) Error: invalid array width!
Uncaught Error: invalid array width!
```

If I count the rows I get an answer:

![property1](https://cloud.githubusercontent.com/assets/6749519/17694530/2b8faefe-639c-11e6-9cdc-fc3f604135a8.png)

**Comment by jonahwh** [comment]
I have this same issue, but not with a large number of columns, but with just a large number of results, it seems. Hitting the "Cancel" button causes the results to be displayed.

**Comment by camsaul** [comment]
This is probably related to #664

**Comment by tlrobinson** [comment]
@danje @jonahwh We believe this has been fixed in master. If possible could you try it out and confirm.

You can build from master, use the metabase/metabase-head:latest Docker image, or this jar: https://9000-30203935-gh.circle-artifacts.com/6/home/ubuntu/metabase/target/uberjar/metabase.jar

**Comment by tlrobinson** [comment]
I'm closing this as we believe it's fixed and no one here or in #3407, #3687, or #3751 has said otherwise, but if you find it's not fixed in 0.21 let us know.

**Comment by aonic** [comment]
Confirmed this is fixed
